test/REGRESSIONS

```diff
@@ -686,6 +679,668 @@ Reviewed 2015-01-23
 ===========================
 
 
+===============================
+cray-module-correctness-cray-x?
+Inherits 'general'
+Reviewed 2015-01-29
+===============================
+
+nonblocking comm interface update needed - Greg
+-----------------------------------------------
+build failed in commDiagnostics (2015-01-30..)
+
+
+=== sporadic failures below ===
+
+Occasional "unexpected termination of the channel"
+--------------------------------------------------
+any
+
+
+==========================================
+cray-module-correctness-cray-x?-none*
+Inherits 'cray-module-correctness-cray-x?'
+Reviewed 2015-01-29
+==========================================
+
+
+===========================================
+cray-module-correctness-cray-x?-gasnet*
+Inherits 'cray-module-correctness-cray-x?'
```

Comment:
Similarly, I'd expect any multi-locale configurations to inherit from either 'gasnet*' or 'no-local'.